src/NonlinearSolve.jl

```diff
@@ -10,6 +10,7 @@ using ForwardDiff: Dual
 using LinearAlgebra
 using StaticArraysCore
 using RecursiveArrayTools
+import EnumX
```

Comment:
missing from the Project.toml

Reply:
yeah i added the pkg in the env but didn't commit it yet. will do it in the next commit